test/javascript/components/iteration_summary.test.js

```diff
@@ -0,0 +1,20 @@
+// Deps
+import React from "react";
+
+// Test deps
+import { render } from "@testing-library/react";
+import "@testing-library/jest-dom/extend-expect";
+
+// Component
+import { IterationsSummaryTable } from "../../../app/javascript/components/iteration_summary.jsx";
+
+test("renders component", () => {
+  const { container, getByText } = render(
+    <IterationsSummaryTable
+      solutionId={1}
+      iterations={[{ id: 1, testsStatus: 'passed' }]}
+    />
+  );
+
+  expect(getByText("1: passed")).toBeInTheDocument();
+});
```

Comment:
This is really just a stub to have jest working up. We can use jest to check any unit level things (for fast iteration), but my preference is to just do end-to-end testing of components via Rails system tests (below).

Reply:
>  end-to-end testing of components

I recommend.